x/cns/types/types.go

```diff
@@ -1 +1,5 @@
 package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+var DefaultFee = sdk.NewCoin("test", sdk.NewInt(100000))
```

Comment:
How about turning this into a function that takes a denom as parameter?

```golang
DefaultFee = func(denom string) { return sdk.NewCoin(denom, sdk.NewInt(100000)) }
```

Reply:
yes, this will be a genesis param in the end. Will be updating this along with genesis state.